src/useClickAway.ts

```diff
@@ -8,10 +8,14 @@ const useClickAway = (
   onClickAway: (event: KeyboardEvent) => void,
   events: string[] = defaultEvents
 ) => {
+  const savedCallback = useRef(onClickAway);
```

Comment:
Instead of saving the callback inside the `useClickAway` hook, isn't it better if consumer do it manually?

```js
const onClickAway = useCallback(() => { /* ... */ });
useClickAway(ref, onClickAway);
```

Reply:
I don't want to write extra code on every `useClickAway` usage
and this approach is less effective

Reply:
may I ask what is the purpos of using the useRef here ?
why not simply use the callback ?